packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -626,10 +656,12 @@ function MultiAutocomplete<T extends Item>({
   return (
     <Autocomplete
       {...props}
-      isMulti
+      type="single"
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this seems odd. Shouldn't it be `multi`?

```
type="multi"
```

Reply:
This is correct, The `MultiAutocomplete` component uses the `SingleAutocomplete` internally which clears the selection on every select. I've renamed the `isMulti` to `clearOnSelect`.